### PR TITLE
Update `react-aria`

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Comment by sebald** [comment]
Does this also include the aria update?

**Comment by OsamaAbdellateef** [comment]
> Does this also include the aria update?

yes it does 
i just updated component stuff

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@babel/core": "7.21.8",
     "@changesets/changelog-github": "0.4.8",
     "@changesets/cli": "2.26.2",
+    "@internationalized/date": "3.3.0",
```
Comment (sarahgm) [inline package.json:15]: why does it have to be on root package-json?

**Comment by sebald** [comment]
@OsamaAbdellateef conflicts 🙈

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef conflicts see_no_evil
We can merge now before any new conflicts xD

**Review by sebald — APPROVED** [review]

**Pull request merged**